internal/pkg/service/stream/storage/level/local/slice.go

```diff
@@ -25,18 +25,17 @@ type Slice struct {
 	AllocatedDiskSpace datasize.ByteSize `json:"allocatedDiskSpace"`
 }
 
-func (f File) NewSlice(path string, prevSliceSize datasize.ByteSize) (Slice, error) {
+func (f File) NewSlice(path string) (Slice, error) {
 	// Create filename according to the compression type
 	filename, err := compression.Filename("slice.csv", f.Compression.Type)
 	if err != nil {
 		return Slice{}, err
 	}
 
 	return Slice{
-		Dir:                filepath.Join(f.Dir, NormalizeDirPath(path)),
-		Filename:           filename,
-		Compression:        f.Compression,
-		DiskSync:           f.DiskSync,
-		AllocatedDiskSpace: f.DiskAllocation.ForNextSlice(prevSliceSize),
```

Comment:
`AllocatedDiskSpace` field will be set by the `statistics` domain, to keep in the separation of individual domains and the they connection via the `plugin` pkg.